File: benchmark/results/ts-invocation-script/run-004/b-baseline/output/src/domain/analytics/services/MetricsAggregator.ts
import { Event } from "../aggregates/Event";

/** MetricsAggregator domain service. */
export class MetricsAggregator {
  create(input: Partial<Event>): Event {
    void input;
    throw new Error("not implemented");
  }

  findById(id: string): Event | null {
    void id;
    throw new Error("not implemented");
  }

  list(limit: number): Event[] {
    void limit;
    throw new Error("not implemented");
  }

  delete(id: string): void {
    void id;
    throw new Error("not implemented");
  }
}

File: benchmark/results/java-invocation-inline/run-001/b-baseline/output/com/metaengine/demo/analytics/services/MetricsAggregator.java
package com.metaengine.demo.analytics.services;

import com.metaengine.demo.analytics.aggregates.Event;
import java.util.List;

/** Domain service that rolls events up into aggregated metrics. */
public class MetricsAggregator {
    /** Aggregate metrics from the event described by the partial input. */
    public Event create(Event input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Look up an aggregated event by id. Returns null when not found. */
    public Event findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List up to {@code limit} aggregated events. */
    public List<Event> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Remove aggregated metrics for the given id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}
